newscoop/library/Newscoop/Services/AuditService.php

```diff
@@ -40,14 +40,13 @@ public function __construct(EntityManager $em, UserService $userService)
      */
     public function update(GenericEvent $event)
     {
+        $user = null;
         try {
             $user = isset($event['user']) ? $event['user'] : $this->userService->getCurrentUser();
-            if (!is_int($user)) {
+            if (!is_int($user) && is_object($user) && method_exists($user, 'getId')) {
```

Comment:
we have no idea what can be passed there by someone else (for example in plugin). I would do interface checking but then we would need one for Legacy User class and entity - don't want to this now.  One line later we call `getId()` so i thing that this check can only prevent other issues.